public/main.html

```diff
@@ -10,24 +10,16 @@
       name="description"
       content="Web site created using create-react-app"
     />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
 
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
     <title>React App</title>
   </head>
   <body class="app">
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <header class="fake-header"></header>
     <header class="header">
       <div class="button-container">
-        <button class="btn-header-back"></button>
-        <button class="btn-header-forward"></button>
+        <button class="btn-header-back btn-header"></button>
```

Comment:
Стоит у кнопки указать тип
```suggestion
        <button type="button" class="btn-header-back btn-header"></button>
```

Это относится ко всем кнопкам

Reply:
указал для всех